### PR TITLE
Component/loading

### PR DESCRIPTION
**Pull request by adCBueno**
image
image

### CODE REVIEW

File: src/components/Loading.tsx [hunk 1]
```diff
@@ -1,15 +1,57 @@
...
+      <ActivityIndicator size={type} color={styles.loading.color} style={[styles.loading, style]} />
     </View>
   )
 }
 
 export default Loading
 
-const styles = StyleSheet.create({})
\ No newline at end of file
+const useStyles = (type: LoadingType) => {
```
[the remark this answers is not in the document]
Comment (adCBueno) [reply src/components/Loading.tsx:29]: In the `ActivityIndicator size={type}` but not anymore in the styles. I should remove it right?

[the remark this answers is not in the document]
Comment (adCBueno) [reply src/components/Loading.tsx]: Done :)

File: package.json [hunk 2]
```diff
@@ -41,4 +37,4 @@
       "node"
     ]
   }
-}
+}
```
[the remark this answers is not in the document]
Comment (adCBueno) [reply package.json:44]: I think it's reverted now

File: src/components/Loading.tsx [hunk 3]
```diff
@@ -1,15 +1,57 @@
...
+
+type LoadingType = 'large' | 'small';
+
+type LoadingProps = {
+  type?: LoadingType;
+}
+
+const Loading = ({
+  type = 'large',
+  style,
```
[the remark this answers is not in the document]
Comment (adCBueno) [reply src/components/Loading.tsx:14]: Added :)

**Comment by gaguayovelez** [comment]
@adCBueno looks good to merge, resolve the conflicts firstly

**Review by gaguayovelez — APPROVED** [review]

**Pull request merged**